app-service-template/go.mod

```diff
@@ -22,10 +22,10 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/diegoholiveira/jsonlogic/v3 v3.2.6 // indirect
 	github.com/eclipse/paho.mqtt.golang v1.4.2 // indirect
-	github.com/edgexfoundry/go-mod-bootstrap/v2 v2.3.0 // indirect
+	github.com/edgexfoundry/go-mod-bootstrap/v3 v2.3.0 // indirect
```

Comment:
Need to run tidy to regenerate this properly

Reply:
done